cedar-policy/src/api.rs

```diff
@@ -299,6 +304,56 @@ impl Authorizer {
     ///
     /// The language spec and Dafny model give a precise definition of how this
     /// is computed.
+    /// ```
+    /// use cedar_policy::{Authorizer,Context,Entities,EntityId,EntityTypeName,
+    /// EntityUid, Request,PolicySet};
```

Comment:
Inconsistent use of spaces between items. Can we run `cargo fmt` on this example?

Reply:
I ran `cargo fmt`. Apparently, it does not format the code in docstring.

Reply:
Unfortunate.  But if you put this in its own file (not in a doc comment) then you can get the formatted output and paste it back here